### DISCUSSION TITLE
Writing custom CSS for some elements properly - need help

### DISCUSSION THREAD

**Discussion by imakeitworknet** [gregsullivan/_tw#121] [Q&A]
Just started using _tw. Works great, but I have a question.

Currently I add some custom CSS to /tailwind/custom/base.css . It all works fine.
But some styling in theme's style.css file look a bit weird, and I can't figure out where it comes from, and how do I change it properly.

For example, most links in the site are underlined by default. The css that makes them underlined is in theme's style.css file, and looks like this:

`
.prose :where(a):not(:where([class~="not-prose"],[class~="not-prose"] *)) {
  color: var(--tw-prose-links);
  text-decoration: underline;
  font-weight: 500;
}
`

How do I properly get rid of that text-decoration in here?
I understand that I could just overwrite/repeat this css in /tailwind/custom/base.css , but somehow I think there needs to be a way not to have this underline in the first place.

**Comment by gregsullivan** [comment]
Hello—Thanks for posting this!

The styles you excerpted are coming from Tailwind Typography. I'd recommend taking a look at [Tailwind Typography's README](https://github.com/tailwindlabs/tailwindcss-typography), at least for a description of what it's meant to do, and then at the [_tw documentation covering Tailwind Typography](https://underscoretw.com/docs/tailwind-typography/).

In short, if you'd like to get rid of the text decoration, you'll want to edit this line:

https://github.com/gregsullivan/_tw/blob/72572d4c9166dca9ba7b4690db320e105e427a87/theme/functions.php#L41

You would most likely want to add `prose-a:no-underline` to the list of styles.

Please let me know if you have any other questions!
